pontoon/base/models.py

```diff
@@ -64,6 +65,12 @@
 UNUSABLE_SEARCH_CHAR = "☠"
 
 
+def get_sentinel_user():
+    return get_user_model().objects.get_or_create(
+        username="pontoon-deleted-user", email="pontoon-deleted-user@mozilla.com"
```

Comment:
We should actually create a [unique](https://bugzilla.mozilla.org/show_bug.cgi?id=1561663#c5) user for each deleted user, so you'll need to create a unique `username` and `email`, e.g. `username=deleted-user-{hash}` and `email=deleted-user-{hash}@mozilla.com`.

Sorry for not making that requirement clearer.

We should also set `first_name` to `Deleted User` to make attributions more readable.

Reply:
Note that we should use the example.com domain name for the email:
https://bugzilla.mozilla.org/show_bug.cgi?id=1646209

Reply:
Updated code with all except hash comment.